openwisp-utils-qa-checks

```diff
@@ -93,17 +96,31 @@ runcheckcommit() {
   fi
 }
 
+runmakemigrationcheck() {
+  if [ ! -f "./tests/manage.py" ]; then
+    echo "File manage.py not found\nSkipping Make Migration Check"
+  else
+    OUTPUT=$(python tests/manage.py makemigrations --dry-run $MODULE)
+    echo $OUTPUT
+    echo $OUTPUT | grep "No changes detected" \
```

Comment:
This line should have redirected output to `/dev/null` to avoid duplicated output:

```shell
echo $OUTPUT | grep "No changes detected" &> /dev/null \
```

But I'm going to fix approve anyway and fix this minor issue on my own. I just wanted to let you know @Weirdo914.